app/models/job.rb

```diff
@@ -0,0 +1,3 @@
+class Job < ApplicationRecord
+  acts_as_taggable_on
```

Comment:
Can you pls add a method similar to this one in here:

https://github.com/OperationCode/operationcode_backend/blob/master/app/models/user.rb#L130-L141

As well as an associated test.  This will confirm that a given job can have tags (plural) added to it, and then we can use this method to query for any jobs that match the search criteria.